src/main/java/com/google/api/generator/gapic/composer/ServiceClientCommentComposer.java

```diff
@@ -254,4 +261,12 @@ private static CommentStatement toSimpleComment(String comment) {
 
     return commentBuilder;
   }
+
+  private static String getSettingsName(String serviceName) {
```

Comment:
These methods are now also unused (as mentioned in the last review), could we remove them as well?

Reply:
Thanks for pointing out.